src/notebooksyncagent.cpp

```diff
@@ -46,17 +48,17 @@
 
 #define NOTEBOOK_FUNCTION_CALL_TRACE FUNCTION_CALL_TRACE(QString("%1 %2").arg(Q_FUNC_INFO).arg(mNotebook ? mNotebook->account() : ""))
 
-static KCalCore::Incidence::Ptr fetchIncidence(const mKCal::ExtendedCalendar::Ptr &calendar, const QString &uid)
+static KCalCore::Incidence::Ptr fetchIncidence(const mKCal::ExtendedCalendar::Ptr &calendar, const KCalId &kcalid)
```

Comment:
Minor detail, but this method is quite similar to KCalCore::Calendar::incidence(). Could just call it directly.